### PR TITLE
feat(iOS): fraudulent website warning setting

### PR DESCRIPTION
**Pull request by sunnylqm**
A Boolean value that indicates whether the webview shows warnings for suspected fraudulent content, such as malware or phishing attempts. (iOS 13+)

### CODE REVIEW

File: docs/Reference.md [hunk 1]
```diff
@@ -1602,6 +1602,15 @@ Whether or not the Webview can play media protected by DRM. Default is false.
 | ------ | -------- | -------- |
 | boolean | No       | Android  |
 
+### `fraudulentWebsiteWarningEnabled`
+
+A Boolean value that indicates whether the web view shows warnings for suspected fraudulent content, such as malware or phishing attemps. The default value is `true`. (iOS 13+)
```
Comment (TheAlmightyBob) [inline docs/Reference.md:1607]: nit:
```suggestion
A Boolean value that indicates whether the web view shows warnings for suspected fraudulent content, such as malware or phishing attempts. The default value is `true`. (iOS 13+)
```

File: src/WebViewTypes.ts [hunk 2]
```diff
@@ -745,6 +746,13 @@ export interface IOSWebViewProps extends WebViewSharedProps {
    * @platform ios
    */
   onCustomMenuSelection?: (event: WebViewEvent) => void;
+
+  /**
+   * A Boolean value that indicates whether the webview shows warnings for suspected
+   * fraudulent content, such as malware or phishing attemps.
```
Comment (TheAlmightyBob) [inline src/WebViewTypes.ts:752]: nit:
```suggestion
   * fraudulent content, such as malware or phishing attempts.
```

**Review by TheAlmightyBob — CHANGES_REQUESTED** [review]
Just minor typos, otherwise looks good. Note that you should expect a delay before this gets merged (and there might be conflicts to resolve), as there are some other things in the queue first.

**Review by TheAlmightyBob — APPROVED** [review]

**Comment by TheAlmightyBob** [comment]
'sup bot, please keep open

**Comment by Titozzz** [comment]
closing for #2994

**Pull request closed without merging**